frontend/appflowy_flutter/lib/user/presentation/screens/sign_in_screen/widgets/third_party_sign_in_buttons.dart

```diff
@@ -30,89 +30,108 @@ class ThirdPartySignInButtons extends StatelessWidget {
 
     if (isMobile) {
```

Comment:
Put the condition inside the `_ThirdPartySignInButton`.

We can use _ThirdPartySignInButton directly here.